lib/pages/App.js

```diff
@@ -277,7 +277,7 @@ export class App extends Component {
       <div className={styles.footerRow}>
         <div className={styles.version}>{`v${version} beta`}</div>
         <div className="mailto">
-          <a className={styles.footerLink} href="mailto:support@tidepool.org?Subject=Feedback on Uploader" target="mailto">Get support</a>
+          <a className={styles.footerLink} href="http://support.tidepool.org/" target="_blank">Get support</a>
```

Comment:
I didn't get any warnings when I tried it. It's from the uploader, which is an app, not an https site.